### DISCUSSION TITLE
New firmware release v1.2.0 with new versioning

### DISCUSSION THREAD

**Discussion by koron** [Yowkees/keyball#358] [Show and tell]
I created a new release for Keyball firmware [v1.2.0](https://github.com/Yowkees/keyball/releases/tag/v1.2.0)
v1.2.0 uses QMK 0.22.x which is supported by [newer Remap](https://remap-keys.app/docs/support-qmk-022)

I changed versioning rule: unified versioning for all keyboards, instead of versioning each keyboards.
Subsequent releases will include zip archives of firmware for each keyboards.
Each zip file contains multiple keymap firmwares for a keyboard.

**Comment by Yowkees** [comment]
非常に長時間にわたり、Keyballファームウェア修正にご尽力いただき本当にありがとうございます！
Keyball界がだんだん巨大になってファームウェアへの期待も大きくなっており、ご協力は本当に心強い限りです。
本当にありがとうございます！
